### ISSUE TITLE
formatOptionLabel doesn't work

### ISSUE THREAD

**Issue by danberger1** [guiyep/react-select-virtualized#56]
From the docs, formatOptionLabel should work as react-select and return a component.
But, when I use it, theres an uncaught error coming from fast-react-select, stating that toLowerCase is not a function

https://github.com/guiyep/react-select-virtualized/blob/933c64153effe28a2372b3d4a545ff2462a05378/src/select-virtualized/shared-components/fast-react-select/helpers/fast-react-select.js

line 35 - 
`formatOptionLabel(item, {}).toLowerCase()`

So, if formatOptionLabel doesn't return just the label, it doesn't work.

For example - 
```
const formatOptionLabel = (option) => {
    return <div>{option.label}</div>
  }
```

That won't work as expected

**Comment by guiyep** [comment]
I will take a look, it may be some regression. Thanks for using the library!

**Comment by guiyep** [comment]
fixed in v2.3.4! thanks for using the library!
